inventory_ha.yml

```diff
@@ -1,24 +1,27 @@
 ---
 all:
   vars:
-    ## EACH OF THE VARIABLES BELLOW CAN BE MOVED TO A GROUPS (all.children.*.vars) OR
-    ## HOSTS (all.children.group.hosts.*) SEcTIONS IN CASE VARS NEED TO DIFFER FOR EACH.
-    ## PRECEDENCE; HOSTS_VARS > GROUPS_VARS > GLOBAL_VARS.
-    # More details: https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#define-variables-in-inventory
+    # Each ssh variables below can be moved to groups (all.children.*.vars) or
+    # hosts (all.children.group.hosts.*) sections if they are different for each
+    # host. precedence; hosts_vars > groups_vars > global_vars. more details:
+    # https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#adding-variables-to-inventory
     ansible_connection: ssh
-    # Comment if no SSH key is used and use `-k` on the command line instead (or `-K`in case sudo requires password)
-    ansible_private_key_file: "/path/ssh_key.dsa"
     ansible_user: root
+    # Comment if no SSH key is in use and add `-k` on the command line instead
+    # (or `-K`in case sudo requires a password)
+    ansible_private_key_file: "/path/to/ssh_key.rsa"
+
   children:
     database:
       hosts:
         db.infra.local:
     repository:
       vars:
-        # Set the contentstore storage details as per the `mount` options
         cs_storage:
           type: nfs
           device: nas.infra.local:/exports/contentstore
+          # as per the `mount` options
+          # https://man7.org/linux/man-pages/man8/mount.8.html#FILESYSTEM-INDEPENDENT_MOUNT_OPTIONS
```

Comment:
type and device both need to stick to the mount "dialectic" so the original location is a better fit. But maybe the comment itself wasn't clear enough to that regard and needs to be clarified